Notify only if window is not in focus

Nanoseb opened the pull request:
Fixes issue #2 . The notification is displayed only when the window in focus is not the one where the long command was running.

I also cleaned up slightly the code, both tabs and spaces were used, so I put spaces everywhere. And using `` ` `` is deprecated so I used `$(..)` instead.

Nanoseb committed:
notify only when unactive window command finishes

Nanoseb committed:
code clean up

remove tab

Nanoseb committed:
improve performances

jb55 reviewed (approved):
I was looking for this. works great!

Commit:
Add support for tmux windows

If a long running command is run from a tmux window, on completion, it
verifies that the tmux window differs before triggering a notification.

Information about the tmux session related to the completed
command is added to the notification to make it easier to figure out in
which tmux window the command was being executed.

Commit:
Support env variable to modify cmd timeout and gnuunits

LONG_RUNNING_COMMAND_TIMEOUT can be set in the environment of the shell.
The same is true for gnuunits which is now called NOTIFYOSD_GNUUNITS.

Change-Id: I546d29ba192424e404e513e9ce069f5cac0bece2

Nanoseb committed:
Merge pull request #1 from madeng/master

Add support for tmux windows and flexible command timeout

Nanoseb committed:
add readme file

Nanoseb committed:
code cleanup and add parameter for sound

Nanoseb committed:
revert env variable name + remove  gnuunit dependency

integration of a function to do the convertion to human readable time

ypid committed:
refactor: Use Tmux variable names instead of cryptic aliases

To improve readability.

ypid committed:
fix: Use Tmux session/window IDs instead of names to handle dup name

The names are still displayed because I think they are still more useful
than the raw IDs.

ypid committed:
chore: Improve README.md style

ypid committed:
refactor: Clean trailing whitespace

ypid committed:
docs: What needs to be installed on Debian for sound support

ypid committed:
fix!: LONG_RUNNING_IGNORE_LIST was ignored; now an array

Nanoseb committed:
Merge pull request #3 from ypid/feat/improve

fix!: LONG_RUNNING_IGNORE_LIST was ignored; now an array and other smaller fixes

Nanoseb committed:
fix .zshrc path
